salt/kubernetes-common/openstack-config.jinja

```diff
@@ -13,6 +13,9 @@ domain-id="{{ pillar['cloud']['openstack']['domain_id'] }}"
 domain-name="{{ pillar['cloud']['openstack']['domain'] }}"
 {% endif %}
 region="{{ pillar['cloud']['openstack']['region'] }}"
+{%- if pillar['system_certificates']['name'] and pillar['system_certificates']['cert'] %}
+ca-file="/etc/pki/trust/anchors/{{ pillar['system_certificates']['name'] }}.crt"
```

Comment:
I'm not sure this is valid? What if my cloud uses a Verisign cert, but I've uploaded a CA for some other purpose?

Reply:
Then question is how are we going design this feature? I not sure for what other purposes uploaded self-signed CA can be used? I'm open for any suggestions

Reply:
K8S should be defaulting to /etc/ssl/ca-bundle.pem - and we should have installed the operator-provided CA into that bundle. Likely, the real issue is in this code.

Reply:
> Then question is how are we going design this feature? I not sure for what other purposes uploaded self-signed CA can be used? I'm open for any suggestions

Operator uploaded system wide CA's will be used by docker, k8s, curl, wget, etc etc etc. Anything that needs to validate a certificate chain. I suspect the issue is that Kubernetes is not reading the system wide bundle.. Maybe it's got a non-SUSE file path or something baked in and we need to update the packaging? or, maybe it has no default at all, and we just need to unconditionally point it at the systemwide bundle?